src/tests/mam4_wet_deposition_unit_tests.cpp

```diff
@@ -0,0 +1,170 @@
+// mam4xx: Copyright (c) 2022,
+// Battelle Memorial Institute and
+// National Technology & Engineering Solutions of Sandia, LLC (NTESS)
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "testing.hpp"
+#include <mam4xx/mam4.hpp>
+
+#include <haero/constants.hpp>
+
+#include <ekat/ekat_type_traits.hpp>
+#include <ekat/logging/ekat_logger.hpp>
+#include <ekat/mpi/ekat_comm.hpp>
+
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <limits>
+#include <memory>
+
+using namespace haero;
+using namespace mam4;
+
+TEST_CASE("test_constructor", "mam4_wet_deposition_process") {
+  ekat::Comm comm;
+  ekat::logger::Logger<> logger("wet deposition constructor test",
+                                ekat::logger::LogLevel::debug, comm);
+  mam4::AeroConfig mam4_config;
+  mam4::WetDepositionProcess::ProcessConfig process_config;
+  mam4::WetDepositionProcess process(mam4_config, process_config);
+  REQUIRE(process.name() == "MAM4 Wet Deposition");
+  REQUIRE(process.aero_config() == mam4_config);
+}
+
+TEST_CASE("test_local_precip_production", "mam4_wet_deposition_process") {
+  ekat::Comm comm;
+  // TODO - figure out how to print this logging information...
+  ekat::logger::Logger<> logger("wet deposition local precip production test",
+                                ekat::logger::LogLevel::debug, comm);
+  int nlev = 72;
+  Real pblh = 1000;
```

Comment:
No idea why these magic numbers, but other tests use them

Reply:
72 levels is the current "standard" number of vertical levels in an atmospheric column in E3SM, so we've been using it to mock up a "typical" column.

The planetary boundary layer /height (which distinguishes between the "near-surface" and "everything else in the atmosphere") doesn't affect most calculations, so we just stick a nice round number in there.